backends/memory/memory_test.go

```diff
@@ -7,6 +7,6 @@ import (
 )
 
 func TestBackend(t *testing.T) {
-	b := New()
-	tester.DoBackendTests(t, b)
+	tester.DoBackendTests(t, New())
+	tester.DoFSWrapperTests(t, New())
```

Comment:
I would fold this into `DoBackendTests`, so that that one always tests everything we would want to test about a backend. It could skip certain tests if the backend does not implement an optional backend.